experimental/class-brand-check-expression.md

```diff
@@ -0,0 +1,19 @@
+# [class.hasInstance](https://github.com/tc39/proposal-class-brand-check)
+
+## ClassExpression
+
+```js
+extend interface ClassExpression {
+    operation: Identifier | null // operation.name currently only has : 'hasInstance'
```

Comment:
I don't think we need to extend `ClassExpression`. For example, the RHS of 
```js
var c = class { }
```
is a class expression, but I don't think
```js
var c = class.hasInstance {}
```
will ever be valid.

In another hand, `class.hasInstance(instance)` is a call expression, if the callee `class.hasInstance` is a class expression, then its AST shape will share with `(class {})()`, though vastly different IMO.

Reply:
Prior art for this:
- `super.abcd()` uses a regular `MemberExpression` (and the `super()` syntax uses a regular `CallExpression`) with a special [`Super`](https://github.com/estree/estree/blob/master/es2015.md#expressions) node in the appropriate location
- `new.target` uses a [`MetaProperty`](https://github.com/estree/estree/blob/18bd9afbe8c348caca82057f1849614d98d6d0b6/es2015.md#metaproperty) with the identifier `new` as the value for the `meta` key.
- `import.meta` uses a [`MetaProperty`](https://github.com/estree/estree/blob/bae1f8084bfa1c45e31d08eb22ef6606e9697a07/es2020.md#metaproperty) with the identifier `import` as the value for the `meta` key.

Reply:
There are some errors in my understanding, indeed `CallExpression` is more suitable. I will correct it later.